src/resolvers/restrictionAccount.resolver.ts

```diff
@@ -25,9 +25,6 @@ export class RestrictionAccountAddressFlagsResolver implements Resolver {
             altText ? altText : 'Select the restriction flags: ',
             choices,
         );
-        if (index < 0 || index > 3) {
```

Comment:
Not aware of which issue is solving this change, could you explain the rationale behind it?